resource_obmcs_core_vnic_attachment_test.go

```diff
@@ -72,6 +75,55 @@ func (s *ResourceCoreVnicAttachmentTestSuite) TestAccResourceCoreVnicAttachment_
 					resource.TestCheckResourceAttrSet(s.VnicResourceName, "private_ip_address"),
 					resource.TestCheckResourceAttr(s.VnicResourceName, "public_ip_address", ""),
 					resource.TestCheckResourceAttr(s.VnicResourceName, "skip_source_dest_check", "false"),
+					func(ts *terraform.State) (err error) {
+						vaId, err = fromInstanceState(ts, s.ResourceName, "id")
+						return err
+					},
+				),
+			},
+			{
+				// Update the VNIC
+				ImportState:       true,
+				ImportStateVerify: true,
```

Comment:
This test looks to be doing `ImportState`, `ImportStateVerify` in each step, perhaps not intentionally.

Reply:
removed all but 2 of these